examples/README.md

```diff
@@ -29,3 +29,27 @@ This ASP.NET web application is a .NET Framework MVC 5 app. It leverages the [co
 
 ## [WebFormApp (.NET Framework)](./DotNetFramework/WebFormApp)
 This ASP.NET web application is a .NET Framework Web Forms app. It demonstrates how to leverage the App Configuration [.NET Standard provider library](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration/) to achieve dynamic configuration and control feature launches with feature flags. The same technique applies to .NET Framework MVC apps.
+
+## [Spring Samples](https://github.com/Azure-Samples/azure-spring-boot-samples/)
+
+Contains multiple Azure App Configuration Spring boot sample projects.
+
+### azure-appconfiguration-sample
+
+This example shows how to use the basic features of Azure App Configuration in a in Java Spring cloud application.
+
+### azure-appconifugration-refresh-sample
+
+This example shows how to dynamically update application settings in a Spring Web application.
+
+### azure-appconfiguration-conversion-sample
```

Comment:
Can we put these two projects _azure-appconfiguration-conversion-sample-initial_ & _azure-appconfiguration-conversion-sample-complete_ in a folder named _azure-appconfiguration-conversion-sample_? It's easier to tell they are the pair of before and after.

Reply:
I don't think so, will verify with the Spring team that they are fine with it. I was copying what I had seen elsewhere.

Reply:
Yes, plesae.

Reply:
This is correct in main already. Will be updated in the default in the next release, but we will link to main as it is the dev branch and will always be the latest changes.